Presentation.Web/app/models/users/userDTO.ts

```diff
@@ -0,0 +1,5 @@
+﻿module Kitos.Models.Users {
+    export interface IUserWithEmailDTO extends Models.Generic.NamedEntity.NamedEntityDTO {
```

Comment:
I tried finding some existing DTO for this, but I couldn't find anything. I only found 1 simple DTO with email, but it was a DPR interface so I didn't want to use it for GlobalAdmin